src/sentry/static/sentry/app/components/commitRow.jsx

```diff
@@ -31,27 +31,39 @@ export default class CommitRow extends React.Component {
     let {id, dateCreated, message, author, repository} = this.props.commit;
     return (
       <PanelItem key={id} align="center">
-        <Flex mr={2}>
+        <AvatarWrapper mr={2}>
           <Avatar size={36} user={author} />
-        </Flex>
-        <Flex flex="1" direction="column">
-          <TruncatedText>{this.renderMessage(message)}</TruncatedText>
-          <div style={{fontSize: 13}}>
+        </AvatarWrapper>
+        <Box flex="1" direction="column" style={{minWidth: 0}} mr={2}>
```

Comment:
I *think* you could also clean up the `flex="1"` since it's not a valid property on `Box`

Reply:
It should be a valid property, no? https://github.com/jxnblk/grid-styled#flex-stringarray.

Reply:
Ahhh, oops, you are right